examples/records/records.bal

```diff
@@ -1,27 +1,50 @@
 import ballerina/io;
 
-// Defines a `record` type named `Coord`.
-type Coord record {
-    int x;
-    string y;
+// Defines a `closed record` type named `Student`. It only allows fields that are specified.
+type Student record {|
+    string name;
+    int age;
+|};
+
+// Defines an `open record` type named `Person`. It allows fields other than those specified.
+type Person record {
+    string name;
+    int age;
 };
 
 public function main() {
-    // Creates a `record`, specifying values for its fields.
-    record { int x; string y; } r = {
-        x: 1,
-        y: "2"
+    // Creates an `open record` by specifying values for its fields.
+    record {string name; int age;} stu = {
+        name: "Harry",
+        age: 12
     };
 
-    // Creates a `Coord` record.
-    Coord c = {
-        x: 1,
-        y: "2"
+    // Accesses the `name` field in `stu`.
+    string name = stu.name;
+    io:println(name);
```

Comment:
Shall we move this just before `io:println(stu == student);`. We need to show first how to create values from `Student` and `Person` first.